### DISCUSSION TITLE
（例）开发新组件 FTree

### DISCUSSION THREAD

**Discussion by Tyh2001** [FightingDesign/fighting-design#5] [Announcements]
> 下面是一个基础的格式，也可以根据自己的习惯以任意的格式编写讨论即可。讨论的作用就是可以证明你现在正在开发 xxx 组件，那么组件遇到问题或者 bug，就可以直接找到你进行修复。

下面是一个简单的模板，仅供参考。

## 功能（可选）

`FTree` 组件将实现用清晰的层级结构展示信息，可展开或折叠。

## 参数（可选）

下面先列举一些必须的配置项，后续会根据实际需求继续添加

| 参数 | 类型  | 说明 |
| ---- | ----- | ---- |
| data | array | 数据 |
| xxx  | xxx   | xxx  |

## 应用场景（可选）

`FTree` 组件将实现处理一些树形结构的数据，实现信息的展示。

## XXX

。。。。。。

**Comment by X-swordx** [comment]
这个组件有人认领了吗，没有我想试试

**Reply by Tyh2001** [reply]
还没，你试试吧 不过这个组件难度系数较高哦

**Reply by Tyh2001** [reply]
@X-swordx 
微信找我一下吧~
